src/foam/u2/view/UnstyledTableView.js

```diff
@@ -459,32 +494,44 @@ foam.CLASS({
                 }).
 
-                forEach(columns_, function([axiomOrColumnName, overrides]) {
-                  var column = typeof axiomOrColumnName === 'string'
-                    ? obj.cls_.getAxiomByName(axiomOrColumnName)
-                    : axiomOrColumnName;
+                forEach(columns_, function([property, overrides]) {
+                  var cls = view.of;
+                  var column;
+                  var obj1 = obj;
+                  if ( typeof property === 'string') {
+                    column = view.columns.find(c => c.name === property);
+                    if ( !column ) {
+                      var columnConfig = this.__context__.columnConfigToPropertyConverter;
+                      if ( !columnConfig ) columnConfig = this.__context__.lookup('foam.nanos.column.ColumnConfigToPropertyConverter').create();
+                      var val = columnConfig.returnPropertyAndObject(view.of, property, obj1);
+                      column = val.propertyValue;
+                      obj1 = val.objValue;
+                    }
+                  } else
+                    column = property;
+                  
                   if ( overrides ) column = column.clone().copyFrom(overrides);
                   this.
                     start().
-                      addClass(view.myClass('td')).
-                      callOn(column.tableCellFormatter, 'format', [
-                        column.f ? column.f(obj) : null, obj, column
-                      ]).
-                      callIf(column.f, function() {
-                        try {
-                          var value = column.f(obj);
-                          if ( foam.util.isPrimitive(value) ) {
-                            this.attr('title', value);
-                          }
-                        } catch (err) {}
-                      }).
-                      call(function() {
-                        if ( column.tableWidth ) {
-                          this.style({ flex: `0 0 ${column.tableWidth}px` });
-                        } else {
-                          this.style({ flex: '1 0 0' });
+                    addClass(view.myClass('td')).
+                    callOn(column.tableCellFormatter, 'format', [
+                      column.f ? column.f(obj1) : null, obj1, column
+                    ]).
+                    callIf(column.f, function() {
+                      try {
+                        var value = column.f(obj1);
+                        if ( foam.util.isPrimitive(value) ) {
+                          this.attr('title', value);
                         }
-                      }).
+                      } catch (err) {}
+                    }).
+                    call(function() {
+                      if ( column.tableWidth ) {
+                        this.style({ flex: `0 0 ${column.tableWidth}px` });
+                      } else {
+                        this.style({ flex: '1 0 0' });
+                      }
```

Comment:
this.style({flex: column.tableWidth ? `0 0 ${column.tableWidth}px` : '1 0 0'});

Reply:
fixed